dbt/dbt-run.py

```diff
@@ -27,18 +27,7 @@ def connection():
 with connection() as conn:
     with conn.cursor() as cur:
         cur.execute(f"""
-            CREATE SCHEMA IF NOT EXISTS 
-            {os.getenv('POSTGRES_SCHEMA')};
-
-            CREATE TABLE IF NOT EXISTS {os.getenv('POSTGRES_SCHEMA')}.{os.getenv('POSTGRES_TABLE')} (
-                "@version" TEXT,
-                "@timestamp" TIMESTAMP,
-                "_id" TEXT,
-                "_rev" TEXT,
-                doc jsonb,
-                doc_as_upsert BOOLEAN,
-                UNIQUE ("_id", "_rev")
-            );
+            CREATE SCHEMA IF NOT EXISTS {os.getenv('POSTGRES_SCHEMA')};
```

Comment:
The schema setup is also being done in the setup file so this can be removed.

Reply:
Oh, i thought i had removed it already. my bad!!

Reply:
Oh, I know why I left this! It's because this script continues to create the _dataemon table within this schema and I didn't want it to crash.